erroranalysis/erroranalysis/_internal/surrogate_error_tree.py

```diff
@@ -246,9 +312,27 @@ def node_to_json(df, tree, nodeid, categories, json,
         "nodeName": node_name,
         "parentId": parentid,
         "parentNodeName": p_node_name,
-        "pathFromRoot": "",
-        "size": float(success + error),
-        "sourceRowKeyHash": "hashkey",
-        "success": float(success)
+        "pathFromRoot": "",  # Note: remove this eventually
```

Comment:
How about creating an issue for it and defining exactly when this can be removed (i.e., what does this depend on?)

Reply:
yes we want to eventually remove it but I need to keep backwards compatiblity for now since we have >= bound on error analysis package.  Perhaps we should start pinning the package.